facade/service_test.go

```diff
@@ -3770,3 +3770,9 @@ func (ft *FacadeIntegrationTest) assertPathResolvesToServices(c *C, path string,
 	c.Assert(foundids, DeepEquals, ids)
 }
 
+func (ft *FacadeIntegrationTest) TestFacade_getChanges(c *C) {
```

Comment:
This test verifies the edge-case where there are no changes.
Can you add a test case that verifies the main part of `getChanges()`?  In other words, something that shows it found an actual change?